ddev/ddev.com
# Auto-format files with prettier, add vscode settings, fixes #281

## The Issue

- https://github.com/ddev/ddev.com/issues/281

## How This PR Solves The Issue

- added `.vscode/settings.json` and `.vscode/extensions.json` to git
- performed `npx prettier --write .` to auto-format all files
- added github action prettier test (`npx prettier --check .`) to test.yml to block wrongly formatted PRs in future
- added new section about Prettier and EditorConfig to README.md

## Manual Testing Instructions

Review rendered at https://pr-format-prettier.ddev-com-front-end.pages.dev/

- check build output and subpages for errors 

## Automated Testing Overview

Maybe use https://ddev.com/blog/ddev-diffy-introduction/ for visual comparison? haven't had time for that yet. https://github.com/garris/BackstopJS is an open source alternative

## Related Issue Link(s)

- https://github.com/ddev/ddev.com/issues/281

## Release/Deployment Notes
